cowbird/services/service.py

```diff
@@ -12,15 +12,13 @@
 SERVICE_PRIORITY_PARAM = "priority"
 SERVICE_URL_PARAM = "url"
 SERVICE_WORKSPACE_DIR_PARAM = "workspace_dir"
-SERVICE_ADMIN_USER = "admin_user"
-SERVICE_ADMIN_PASSWORD = "admin_password"
+SERVICE_ADMIN_USER = "admin_user"  # nosec: B105
```

Comment:
Why those are not declared inside geoserver module as they are not required by any other service?